spec/support/shared_context/message_queue_test_mode.rb

```diff
@@ -0,0 +1,24 @@
+RSpec.shared_context "using the message queue in test mode" do
+  before :all do
+    @config = YAML.load_file(Rails.root.join("config", "rabbitmq.yml"))[Rails.env].symbolize_keys
```

Comment:
Now that this has been extracted, it's probably a good idea to give these variables a more specific name to reduce the change they conflict with variables used in a test.